Bottom: fb9f11469da269f0cf27ad03dada75afb42f75bd
Top:    02f05c9bc7bb6d447810bd3df39b0ed57d0900a8
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-01-07 01:46:18 +0000

lesstif: Use Coord type for line-width, not int


---

diff --git a/src/hid/lesstif/main.c b/src/hid/lesstif/main.c
index d7b5e34..ae0b9fa 100644
--- a/src/hid/lesstif/main.c
+++ b/src/hid/lesstif/main.c
@@ -52,7 +52,7 @@ typedef struct lesstif_gc_struct
 
   Pixel color;
   const char *colorname;
-  int width;
+  Coord width;
   EndCapStyle cap;
   char xor_set;
   char erase;
